packages/smooth_app/test/utils/analytics_helper_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:matomo_tracker/matomo_tracker.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:smooth_app/helpers/analytics_helper.dart';
+
+import '../tests_utils/mocks.dart';
+
+void main() {
+  test(
+    'When user opts out then init Matomo with userId of 0000000000000000',
+    () async {
+      // arrange
```

Comment:
?

Reply:
What exactly causes your confusion?

Reply:
```suggestion
      // init
```

Reply:
> I was just following the arrange-act-assert pattern of unit-testing ;)

Oh that makes sense then! I didn't know the exact terms.